api/v1/types.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1
+
+import (
+	"time"
+)
+
+// Timeseries represents a set of metrics for the same target object
+// (typically a container).
+type Timeseries struct {
+	// Map of metric names to their value.
+	Metrics map[string]Point `json:"metrics"`
```

Comment:
Doesn't this have to be a map from metric names to more than just a single point? A single target-metric pair can have more than one point if the metric has additional labels. For example, if I have one disk-bytes-used data point for device_name=/dev/sda1 and one point for device_name=/dev/sda2, both from the same container, I'm not sure how this schema supports exposing both?